src/components/TextInput/TextInput.spec.tsx

```diff
@@ -123,4 +123,19 @@ describe("Text Input component", () => {
         mount(<StatefulInput disabled={false} />);
         cy.get(TEXT_INPUT_ID).should("not.have.attr", "disabled");
     });
+
+    it("has the autoComplete turned on", () => {
+        mount(<StatefulInput autocomplete={true} />);
+        cy.get(TEXT_INPUT_ID).should("have.attr", "autoComplete", "on");
```

Comment:
Good that HTML is case insensitive 😀
It is lowercase `autocomplete`

Reply:
But know you merged it? 🤔

Reply:
Yes, it is not an issue, it could be `aUtOCoMpLeTe` and still be correct

Reply:
Yep, but should we change it back? I think I had it all lowercase before but changed it without thinking too much just because the IDE suggested it. 🙄